vnext/PropertySheets/JSEngine.props

```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <Import Project="$(SolutionDir)\ExperimentalFeatures.props" Condition="Exists('$(SolutionDir)\ExperimentalFeatures.props')" />
   <PropertyGroup>
+    <JsEnginePropsDefined>true</JsEnginePropsDefined>
+    <!-- Enabling this will (1) Include hermes glues in the Microsoft.ReactNative binaries AND (2) Make hermes the default engine -->
     <UseHermes Condition="'$(UseHermes)' == ''">false</UseHermes>
-    <HermesVersion Condition="'$(HermesVersion)' == ''">0.8.0-staging3</HermesVersion>
+    <!-- This will be true if (1) the client want to use hermes by setting UseHermes to true OR (2) We are building for UWP where dynamic switching is enabled -->
+    <IncludeHermes Condition="'$(IncludeHermes)' == '' And ('$(UseHermes)' == 'true' Or '$(ApplicationType)' == 'Windows Store')">true</IncludeHermes>
```

Comment:
Just thinking worst-case... This looks like a localizable string... Could this be something like "Loja Windows" of "Windows Winkel" on other languages? I.e. folks building on non EN-US machines would fail?

Reply:
And sorry, GitHub is not as flexible as Azure Devops..  I meant to annotate  the string `Windows Store`, not the entire line..

Reply:
I don't believe so, this is not a display string but an app type ID

Reply:
@dannyvv I think the value is not coming from anywhere else but is defined [here](https://github.com/microsoft/react-native-windows/blob/4ba8f260bc6070528244a3472e6bac176eaf012e/vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj#L14) .. We don't care about the value of this props in any other projects ..